web/src/api/axios.ts

```diff
@@ -0,0 +1,33 @@
+import axios, { AxiosInstance } from 'axios';
+
+const instance: AxiosInstance = axios.create({
+  baseURL: 'https://capstone-mock-api.fly.dev',
+  // timeout: 15000,
+});
+
+// 요청 인터셉터 추가하기
+instance.interceptors.request.use(
```

Comment:
So have you tried this out? Wrapping `axios.get()` with try catch and which acts first? If you know about the order of precedence, please share it!